src/mkrelease.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Might be worth adding some comments at the top for how this works for the future :)